src/angular-dayparts.js

```diff
@@ -1,5 +1,16 @@
 angular.module('angular-dayparts', [])
-.directive('angularDayparts', ['$window', '$document', '$timeout', function ($window, $document, $timeout) {
+  .provider('angularDaypartsConfig', function () {
+    this.presetItems = [
+      {"value":"week", "label":"All hours and days"},
+      {"value":"weekend", "label":"Weekends (Sat-Sun)"},
+      {"value":"weekdays", "label":"Weekdays (Mon-Fri)"},
+      {"value":"businessHours", "label":"Business Hours (Mon-Fri, 9am-5pm)"},
+      {"value":"eveningHours", "label":"Evenings (6pm-12pm)"},
+      {"value":"custom", "label":"Custom", disabled: true}
```

Comment:
The demo in this repo is set up for AngularJS `1.6.1`, which means `disabled: true` should be the default ⚡️ .